user/pages/05.blog/2017-05-03-Restart-Detection

```diff
@@ -0,0 +1,13 @@
+---
+title: Restart Detection
+date: 20:50 2017-05-03
+author: Marcel Fuhrmann
+body_classes: header-lite fullwidth blogstyling
+taxonomy:
+    category: wiki
+    tag: [threshold, wiki]
+---
+It's always important to work through your outages and alarm events to get an overview what happened in your system.
+In some cases you won't recognize system restarts because the systems (especially VMs) nowadays do it very fast -- often faster than your ICMP polling cycle.
+
+Would you like to know more? -> [Restart Detection](https://wiki.opennms.org/wiki/Restart_Detection)
```

Comment:
I would add some more explanation to get the readers attention. Something like that:

With SNMP the _hrSystemUptime_ can be used to detect system restarts. The same concept can also be used to detect application restarts within a Java Virtual Machine. The following article describes how OpenNMS can be used to detect system and JVM application restarts using threshold events for audit logs or for notifcations.